app/views/hyrax/base/_work_title.erb

```diff
@@ -1,9 +1,16 @@
 <% presenter.title.each_with_index do |title, index| %>
-    <% if index == 0 %>
-        <h1>
-            <%= title %> <%= presenter.permission_badge %> <%= presenter.workflow.badge %>
-        </h1>
-    <% else %>
-        <h1><%= title %></h1>
-    <% end %>
+  <div class="row">
+    <div class="col-sm-8">
+      <% if index == 0 %>
+        <h4><%= title %>
```

Comment:
Why is this heading h4 here and h1 on line 9?  Why did we go to h4?  Is there already h1-h3 on the page?  The heading level is important for accessibility.

Reply:
I'm switching this up to h2 and the work_type to h1, I think this is probably what we need for accessibility.